src/main/java/be/seeseemelk/mockbukkit/configuration/ServerConfiguration.java

```diff
@@ -49,6 +51,13 @@ public ServerConfiguration()
 		ticksPerSpawn.put(SpawnCategory.WATER_UNDERGROUND_CREATURE, 1);
 		ticksPerSpawn.put(SpawnCategory.WATER_ANIMAL, 1);
 		ticksPerSpawn.put(SpawnCategory.AMBIENT, 1);
+
+		spawnLimits.put(SpawnCategory.MONSTER, 70);
```

Comment:
Would it not be better if MetaMiner fetched these default values (same in WorldMock)? (i.e load the values from a json file)

Reply:
This is mostly me overthinking stuff, let's not